src/CommunityToolkit.Maui/Views/RangeSlider/RangeSlider.shared.cs

```diff
@@ -0,0 +1,437 @@
+using System.ComponentModel;
+using CommunityToolkit.Maui.Core;
+using Microsoft.Maui.Controls.Shapes;
+
+namespace CommunityToolkit.Maui.Views;
+
+/// <summary>RangeSlider control.</summary>
+public partial class RangeSlider : ContentView
+{
+	/// <summary>The backing store for the <see cref="MinimumValue" /> bindable property.</summary>
+	public static readonly BindableProperty MinimumValueProperty = BindableProperty.Create(nameof(MinimumValue), typeof(double), typeof(RangeSlider), defaultValue: RangeSliderDefaults.MinimumValue);
```

Comment:
could you refactor this bindable properties to use our SourceGenerator?
If you need a reference, you can look here

- https://github.com/CommunityToolkit/Maui/pull/2964